source/administrator-guide/configuring-the-catalog/portal-configuration.rst

```diff
@@ -0,0 +1,142 @@
+
+A sub portal is defined by:
+
+.. figure:: img/portal-subportal.png
+
+
+When creating a subportal with an identifier ``ìnspire`` for example, a new entry point on your catalogue will be available:
+http://localhost:8080/geonetwork/inspire/. Accessing the catalogue through it will only provide access to records
+matching the filter defined for this portal. The ``search filter`` parameter value use the Lucene query parser syntax
+(see http://lucene.apache.org/java/2_9_1/queryparsersyntax.html) and is applied to all searches.
+
+
+User privileges apply as in the main instance.
+
+
+The logo and name of the portal will be displayed instead of the main instance information:
+
+.. figure:: img/portal-header.png
+
+
+A CSW service is also available for this portal http://localhost:8080/geonetwork/inspire/eng/csw (and replace the virtual CSW feature).
+
+
+A portal can also use a specific user interface configuration.
+
+
+The list of portal available is at http://localhost:8080/geonetwork/srv/api/sources
+
+.. figure:: img/portal-list.png
+
+
+Example of usage
+````````````````
+
+Creating an INSPIRE directive space
+-----------------------------------
+
+For the INSPIRE directive, catalogue administrator needs to publish an entry point providing access only to INSPIRE related records. An INSPIRE subportal can be created with a filter on keywords ``+thesaurusName:"GEMET - INSPIRE themes, version 1.0"``.
+
+
+
+Creating a space for partners
+-----------------------------
+
+Some organizations are opening the catalogue to a set of partners. In such case, each partner access the catalogue
+and create their records in dedicated groups usually. A good example is https://www.geocat.ch/
+providing a main search filter ``catalogue``.
+
+.. figure:: img/portal-geocatch.png
+
+The concept of sub portal adds the possibility to create dedicated URL for each partner. The header can contain
+the partner identification with name and logo and optionnaly the user interface can also be customized
+(see :ref:`user-interface-configuration`
+
+To setup this kind of configuration, the basic principle is to have:
+
+- One group for each partner with one or more users
+
+- One portal for each partner with a filter matching records in that group
+
+
+
+To configure this, apply the following steps:
+
+- Create a group for the partner eg. ``oca`` (see :ref:`creating-group`).
+
+- Create at least one user for the partner (see :ref:`creating-user`). User created must be member of the group ``oca``. If you want user to be able to configure the portal (eg. change the name, choose a logo), user must have at least the ``UserAdmin`` profile for the group ``oca``.
+
+- Create a portal. This portal can have the same name. eg. ``oca``. The portal filter can be based on the fact that a record published in the group ``oca`` is in this portal. Such filter is ``+_groupPublished:oca``. Once created the portal is accessible at http://localhost:8080/geonetwork/oca.
+
```

Comment:
Create a sub-portal. This can have the same name as the group, eg. ``oca`` but this is not essential. The filter can be created using the fact that a record published in the group ``oca`` should be in this sub-portal, using the syntax ``+_groupPublished:oca``. Once created the sub-portal